src/support/z_pivot_rsc.erl

```diff
@@ -141,11 +141,12 @@ queue_all(Context) ->
         end.
 
 %% @doc Insert a rsc_id in the pivot queue
+-spec insert_queue(m_rsc:resource(), #context{}) -> ok | {error, eexist}.
```

Comment:
The spec in line 144 uses a record, please define a type for the record and use that instead.